internal/planner/planner.go

```diff
@@ -16,6 +19,8 @@ var PLANNERS = []plansdk.Planner{
 // Return a merged shell plan from shell planners if user defined packages
 // contain one or more dev packages from a shell planner.
 func GetShellPlan(srcDir string, userPkgs []string) *plansdk.ShellPlan {
+	defer trace.StartRegion(context.Background(), "getShellPlan").End()
```

Comment:
does it matter that the context is different?

Reply:
The context groups regions into a single task, making it easier to see the overall timeline. This region will still be visible on its own, although it would be a bit nicer to use the same context. It was just a bit trickier to propagate the context into this function without changing the Devbox interface.